query-node/schemas/content.graphql

```diff
@@ -70,6 +70,9 @@ type Channel @entity {
   "List of channel collaborators (members)"
   collaborators: [Membership!]
```

Comment:
Also, I think we no longer need a separate `collaborators` field, collaborator members are already part of `ChannelAgentPermissions` entity; also better to rename `ChannelAgentPermissions` to `Collaborators`.

Reply:
Done. I just hope it doesn't reduce querying possibilities. In case it does, the redundancy can be reintroduce by reverting https://github.com/Joystream/joystream/pull/4150/commits/84f912d597294a8531fd9a9538c81c95966ba91b.